src/components/calculators/PropertyTaxes.vue

```diff
@@ -51,8 +51,24 @@ Property Taxes covers both Stamp Duty and Land  & Buildings Transaction Tax ( LB
                 <input name="propertyValue" v-validate="'required|decimal:2'" type="number" min="1" step="0.01" v-model.number="input.propertyValue" class="form-item__input form-item__input--number" placeholder="Enter your property value here"/>
               </div>
             </fieldset>
+            <fieldset class="form-item">
+              <p class="form-item__title">Tax year:</p>
+              <div class="form-item__wrapper form-item__wrapper--year-selector flex-wrap">
+                <label class="form-item__label flex-grow" :class="{active: input.date === '2018'}">
+                  <input type="radio" v-model="input.date" value="2018" class="form-item__input form-item__input--radio form-item__input--radio--hidden"/>2018 Tax Year Calculation
```

Comment:
Check with the content team whether this is the right label in the stamp duty cae